FFOSBR.js

```diff
@@ -12,67 +12,136 @@
 
 },{"./settings":2}],2:[function(require,module,exports){
 function Settings() {
-  var timeInMilliSec = 0;
-
-  var options = {
-    photos: false,
-    videos: false,
-    contacts: false,
-    text: false,
-    intervalTime: 0, // pass in value in hours
+
+  this.options = {
+    photos: true,
+    videos: true,
+    contacts: true,
+    text: true,
+    intervalTime: 24, // pass in value in hours
     id: 0,
     registeredTimer: false,
-    repeat: false
+    repeat: true
   };
 
+  // Load options if present
+  var retrievedOptions = localStorage.getItem('ffosbrOptions');
+
+  if (retrievedOptions !== null) {
+    this.options = JSON.parse(retrievedOptions);
```

Comment:
Should we be doing any sort of validation here to ensure that what we parsed was a valid options dictionary?